src/main/cliapp/src/containers/dataLoadsPage/DataLoadsComponent.js

```diff
@@ -45,7 +45,7 @@ export const DataLoadsComponent = () => {
 	const [bulkLoadDialog, setBulkLoadDialog] = useState(false);
 	const [expandedGroupRows, setExpandedGroupRows] = useState(null);
 	const [expandedLoadRows, setExpandedLoadRows] = useState(null);
-	const [expandedFileRows, setExpandedFileRows] = useState(null);
+	//const [expandedFileRows, setExpandedFileRows] = useState(null);
```

Comment:
Should that be removed? Or is that commenting out only temporary?

Reply:
ya the tables need to get reworked, in order to support to new structure.